Remotion/Data/DomainObjects/DateTimeStorageType.cs

```diff
@@ -0,0 +1,16 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Data;
+
+namespace Remotion.Data.DomainObjects;
+
+/// <summary>
+/// Holds the storage type to be used with the <see cref="DateTimeStorageTypeAttribute"/> to define
+/// the rdbms storage type of a DateTime property.
+/// </summary>
+public enum DateTimeStorageType
```

Comment:
```suggestion
/// <seealso cref="DateTimeStorageTypeAttribute"/> 
public enum DateTimeStorageType
```
With the other PR, please add the service, too, as a seealso.